less/common.less

```diff
@@ -113,6 +122,11 @@ html.no-js .no-js-warning {
   border-top: 0.2rem solid @white;
 }
 
+#content {
+  padding-left: 0;
+  padding-right: 0;
+}
```

Comment:
Argh, I tried the following on that full width coloured section

```
margin-left: -(@grid-gutter-width / 2);
padding-left: (@grid-gutter-width / 2);
```
This leaves a gap to the right due to the negative offset on the left.

Adding `margin-right: -(@grid-gutter-width / 2);` simply just makes the section wider but the gap is still there, which introduces a horizontal scrollbar to the page.

Also, this approach won't work on larger screen either as we can't specify how much offset we want to move. (see https://github.com/mmmavis/teach.webmaker.org/blob/develop/less/components/page.less#L21-L23)

:sob: